init.sh

```diff
@@ -71,19 +73,33 @@ oc env dc/jenkins KIE_SERVER_USER=${KIE_SERVER_USER} KIE_SERVER_PASSWORD=${KIE_S
 echo
 echo "Waiting for Jenkins build to begin..."
 echo
-wait_for_running_first_build "jenkins" "${OSE_CI_PROJECT}"
+wait_for_running_build "jenkins" "${OSE_CI_PROJECT}"
+
+# Cancel initial build since this is a binary build with no content
+oc cancel-build jenkins-1
+
+oc start-build jenkins --from-dir="${SCRIPT_BASE_DIR}/docker/jenkins"
+
+wait_for_running_build "jenkins" "${OSE_CI_PROJECT}" "2"
 
-oc build-logs -f jenkins-1
+oc build-logs -f jenkins-2
 
 # Process Nexus Template
-oc process -v APPLICATION_NAME=nexus,GIT_REF=${GIT_REF} -f "${SCRIPT_BASE_DIR}/support/templates/nexus-template.json" | oc create -f -
+oc process -v APPLICATION_NAME=nexus -f "${SCRIPT_BASE_DIR}/support/templates/nexus-template.json" | oc create -f -
 
 echo
 echo "Waiting for Nexus build to begin..."
 echo
-wait_for_running_first_build "jenkins" "${OSE_CI_PROJECT}"
+wait_for_running_build "nexus" "${OSE_CI_PROJECT}"
+
+# Cancel initial build since this is a binary build with no content
+oc cancel-build nexus-1
+
+oc start-build nexus --from-dir="${SCRIPT_BASE_DIR}/docker/nexus"
+
+wait_for_running_build "nexus" "${OSE_CI_PROJECT}" "2"
 
-oc build-logs -f nexus-1
+oc build-logs -f nexus-2
```

Comment:
you can just do oc logs bc/${build-config-name} -f  so you don't have to rely on the number